src/main/java/kidsnd274/duke/TaskList.java

```diff
@@ -0,0 +1,115 @@
+package kidsnd274.duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import kidsnd274.duke.taskobjects.Task;
+
+/**
+ * TaskList object that is just a container for the task list
+ */
+public class TaskList {
+    ArrayList<Task> taskList;
+
+    /**
+     * Default constructor for TaskList
+     *
+     * Takes in no argument and creates an empty task list
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Alternative constructor for TaskList
+     *
+     * Takes in an {@code ArrayList<Task>} and sets it as a task list
+     * @param taskList The task list that is this new instance wraps around
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Adds a new {@code Task} to the task list
+     * @param task Provided Task that is added to the task list
+     * @return Returns true is addition is successful, else false
```

Comment:
should have a blank line between description and parameter section.

The javadocs for all your methods in `TaskList.java` seems to have this problem.